functions/New-DbaDatabase.ps1

```diff
@@ -0,0 +1,317 @@
+function New-DbaDatabase 
+{
+  <#
+.SYNOPSIS 
+New-DbaADatabase creates a new database
+
+.DESCRIPTION
+New-DbaDatabase creates a new database with a single user filegroup, and the PRIMARY filegroup reserved for system objects.
+It allows creation with multiple files, and sets all growth settings to be fixed size rather than percentage growth.
+
+.PARAMETER SqlInstance
+The SQL server instances to be connected to; can be passed in via the pipeline.
+
+.PARAMETER Force
+The force parameter will ignore some errors in the parameters and assume defaults.
+
+.PARAMETER WhatIf 
+Shows what would happen if the command were to run. No actions are actually performed. 
+
+.PARAMETER Confirm 
+Prompts you for confirmation before executing any changing operations within the command. 
+
+.PARAMETER Silent 
+Use this switch to disable any kind of verbose messages
+
```

Comment:
- Lots of parameters missing from this help
 - A minimal example would be nice ("What is the minimum information you _must_ pass in order for something to happen?")
 - An example showing pipeline usage would be great